app/builders/application_builder.rb

```diff
@@ -59,18 +61,21 @@ def dependent_attributes(online_application)
   end
 
   def online_applicant_attributes(online_application)
-    fields = %i[title first_name last_name date_of_birth ni_number married]
+    fields = [:title, :first_name, :last_name, :date_of_birth, :ni_number, :married]
```

Comment:
@garytaylor I don't mind C but I think the % notation in Ruby was inspired from Perl which makes it even worse to me :)

So, if everyone is happy I will update the rubocop config to disable all the % notations and run the `rubocop auto fix` to update the rest of the code.